patterns/behavioral/command.py

```diff
@@ -1,61 +1,71 @@
 """
+Command pattern decouples the object invoking a job from the one who knows
+how to do it. As mentioned in the GoF book, a good example is in menu items.
+You have a menu that has lots of items. Each item is responsible for doing a
+special thing and you want your menu item just call the `execute` method when
+it is pressed. To achieve this you implement a command object with the `execute`
+method for each menu item and pass to it.
+
 *TL;DR
-Encapsulates all information needed to perform an action or trigger an event.
+Object oriented implementation of callback functions.
 
 *Examples in Python ecosystem:
 Django HttpRequest (without `execute` method):
- https://docs.djangoproject.com/en/2.1/ref/request-response/#httprequest-objects
+https://docs.djangoproject.com/en/2.1/ref/request-response/#httprequest-objects
 """
 
-import os
 
```

Comment:
I'm also not a fan of interfaces in Python. They are not required so I recommend removing the `Command` class.

Reply:
I do remove it!

But to know it myself, what are usages of interfaces in python?

Is not it to force the developer to implement 'execute' like methods?

Reply:
If you want to be strict and format about what needs to be implemented in a subclass to conform to an interface, you can use the `abc` module. But IMHO such a simple class with a single function defeats that purpose, so it's better to skip it completely and prefer convention.